plan_fixtures_test.go

```diff
@@ -264,7 +264,7 @@ var PlanTestFixture6 = &PlanTestFixture{
 }
 
 
-func (f *PlanTestFixture) Plan(t *testing.T, query string) *QueryPlan {
+func (f *PlanTestFixture) Plan(t *testing.T, query string) (*QueryPlan, error) {
```

Comment:
I've broken these test operations a bit more to add an additional check for error outcomes.